### DISCUSSION TITLE
Modal Size should accept percentage is better than fixed size options.

### DISCUSSION THREAD

**Discussion by spencerchang** [heroui-inc/heroui#1306] [Feedback]
Modal Size should accept percentage is better than fixed size options. 
Version 1 accept percentage to make the size but version 2 only accept fixed options.
Sometime 5xl is not big enough but i don't want the full screen size.

Thank you.

**Comment by jrgarciadev** [comment]
Hey you can pass any max size you like as follows:


```tsx

<Modal
  className="max-w-[60%]"
>
 ....
 </Modal>

```
